timesketch/api/v1/resources/user.py

```diff
@@ -186,7 +186,9 @@ def post(self, sketch_id):
                 sketch.revoke_permission(permission=permission, group=group)
 
         public = form.get("public")
-        if public == "true":
+        # TODO: Remove string check. Non-pythonic check is needed because the old UI
+        # returns a string of true or false and not a boolean.
+        if public is True or public == "true":
```

Comment:
is there a chance that this would be `True` instead of `true`? If so we might want to check for `public.toLower() == "true"` WDYT?

Reply:
No, this is set from the backend and not by the user. It hasn't changed in years. I opt to keep this as is because we are deprecating the old stuff soon.